robottelo/common/constants.py

```diff
@@ -169,7 +169,9 @@
     ('rhel', 'rhva6', 'rhva6S', 'repo_ver', '6Server')
 ]
 
+#: Name (not label!) of the default organization.
```

Comment:
That would be nice. I've also added this docstring because it is referenced from elsewhere in the documentation. Adding a docstring for this chunk of data makes the reference link correctly.